src/main/java/org/icgc/argo/program_service/controller/ProgramController.java

```diff
@@ -319,4 +319,12 @@ public ResponseEntity<AddInstitutionsResponseDTO> addInstitutions(
     }
     return new ResponseEntity(addInstitutionsResponseDTO, HttpStatus.OK);
   }
+
+  @GetMapping(value = "/dataCenter")
+  public ResponseEntity<List<DataCenterDTO>> listDataCenters(
+      @Parameter(hidden = true) @RequestHeader(value = "Authorization", required = true)
+          final String authorization) {
+    val dataCenterEntities = serviceFacade.listDataCenters();
+    return new ResponseEntity(dataCenterEntities, HttpStatus.OK);
+  }
 }

```

Comment:
Should we have a new controller for data centers?

Reply:
Moved the endpoint to DataCenterController